src/tests/functional/plugin/conformance/subgraphs_dumper_new/include/utils/node.hpp

```diff
@@ -167,6 +168,48 @@ inline std::string get_node_type(const std::shared_ptr<ov::Node>& node) {
     return "static";
 }
 
+static std::map<std::string, std::string> get_max_ops_versions() {
+    std::map<std::string, std::set<std::string>> unique_ops = FuncTestUtils::get_unique_ops();
+    std::map<std::string, std::string> max_ops_versions;
+
+    size_t priority = 1;
+    for (auto op_info : unique_ops) {
+        std::vector<int> available_opsets;
+        for (auto opset : op_info.second) {
+            available_opsets.push_back(std::atoi(opset.c_str()));
+        }
+        int max_opset = *std::max_element(available_opsets.begin(), available_opsets.end());
+
+        max_ops_versions.insert({op_info.first, std::to_string(max_opset)});
+    }
+    return max_ops_versions;
+}
+
+//  { op_name, max_opset_version }
+static std::map<std::string, std::string> max_ops_versions = ov::tools::subgraph_dumper::get_max_ops_versions();
```

Comment:
we can move this definition inside `get_node_priority_by_version()`. Looks like we did not use it in other places

Reply:
moved